clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ElasticSearchServerGroupTagger.java

```diff
@@ -86,7 +86,16 @@ public void alert(String cloudProvider,
                                                String tagName,
                                                int maxResults) {
     return entityTagsProvider.getAll(
-      cloudProvider, ENTITY_TYPE, null, null, accountId, null, null, Collections.singletonMap(tagName, "*"), maxResults
```

Comment:
no change here ... just took it upon myself to split an overly long line of code